### DISCUSSION TITLE
Make an app always be zoom-fullscreen

### DISCUSSION THREAD

**Discussion by Wes974** [asmvik/yabai#1827] [Q&A] [answered]
Hi,

Quick question, I would like to make Safari and every window that I may open always be in `zoom-fullscreen`. I don't want to always select the window and press `alt + f` (mapped to `yabai -m window --toggle zoom-fullscreen`) all the time.

I tried to look in the documentation for `yabai -m rule` but I didn't find something that I could use, or maybe I just missed it.
But there was something about grid is this the best way or am I missing something ?

Could anyone enlighten me please ?

Thanks

**Comment by Bellavene** [comment]
Set Safari to float and make a signal for new Safari windows set to yabai -m window --grid 1:1:0:0:1:1

**Comment by asmvik** [comment, marked as answer]
yabai -m rule --add app="^Safari$" manage=off grid=1:1:0:0:1:1
